style.css

```diff
@@ -0,0 +1,193 @@
+    --black: #000000;
+    --white: #ffffff;
+    --light-orange: #9f5757;
+    --blue-black: #273852;
+    --light-gray: #aaaaaa;
+    --orange: #d35b44;
+}
+
+html, body{
+    margin: 0;
+    padding: 0;
+}
+
+body{
+    height: 100vh;
+    font-family: 'Roboto', sans-serif;
+}
+
+.d-flex{
+    display: flex;
+}
+
+.row{
+    flex-direction: row;
+}
+
+.col{
+    flex-direction: column;
+}
+
+.center{
+    justify-content: center;
+    align-items: center;
+}
+
+header{
+   background-color: var(--blue-black);
+   width: 100%;
+}
+
+header > div{
+    text-transform: uppercase;
+    margin: 20px;
+    background-color: var(--white);
+    font-weight: bold;
+    border: 2px solid var(--light-orange);
+    padding: 12px;
+}
+
+#title{
+    font-size: 2rem;
+    margin: 0px 20px 2px 20px;
+}
+
+#sub-title{
+    font-size: small;
+}
+
+main{
+    flex: 1 0 auto;
+    margin: 24px 28px;
+}
+
+#articles-header{
+    justify-content: space-between;
+    align-items: center;
+    padding-bottom: 24px;
+    border-bottom: 1.5px solid var(--black);
+}
+
+#category > label, #subscribe-form > label{
+    text-transform: uppercase;
+    font-size: small;
+    font-weight: bolder;
+    margin-bottom: 8px;
+}
+
+#filter, #text-subscribe{
+    text-transform: capitalize;
+    padding: 4px 12px;
+    height: 34px;
+    width: 14rem;
+    border: 1px solid var(--black);
+}
+
+#main-title{
+    font-size: 1.6rem;
+    font-weight: bold;
+    text-transform: uppercase;
+}
+
+#text-subscribe{
+    height: 24px !important;
+}
+
+#btn-subscribe{
+    height: 2.12rem;
+    background-color: var(--black);
+    color: var(--white);
+    border: var(--black);
+    cursor: pointer;
+}
+
+.article-container{
+    width: 70%;
+}
+
+.broder-bottom{
+    padding-bottom: 24px;
+    border-bottom: 1px solid var(--light-gray);
+}
+
+.article-image{
+    width: 300px;
+    height: 140px;
+    background-color: var(--light-gray);
+    margin-right: 24px;
+    margin-top: auto;
+}
+
+.article-title{
+    font-size: large;
+    font-weight: bolder;
+    margin-bottom: 2px;
+}
+
+.article-desc{
+    font-size: small;
+}
+
+.article-body > small{
+    font-size: 0.7rem;
+}
+
+.article-body > button{
+    background-color: var(--orange);
+    border: 1px solid var(--orange);
+    height: 2.5rem;
+    padding: 1px 16px;
+    color: var(--white);
+}
+
+footer{
+    flex-shrink: 0;
+    background-color: var(--blue-black);
+    height: 46px;
+    color: var(--white);
+}
+
+@media screen and (max-width: 768px) {
+    #articles-header{
+        flex-direction: column !important;
+        padding-bottom: 24px;
+        border-bottom: 1.5px solid var(--black);
+    }
+
+    #category, #subscribe-form{
+        justify-content: center;
+        align-items: center;
+        flex-direction: column-reverse;
+        margin-top: 10px;
+    }
+
+    label{
+        margin-top: 8px;
+    }
+
+    #articles-header :nth-child(1) { order: 2; }
+    #articles-header :nth-child(2) { order: 1; }
```

Comment:
why order values are not matching with children's index position?

Reply:
For mobile screen we need to switch the order for the children.